app/routes/people/person/personal-information.js

```diff
@@ -11,11 +17,99 @@ export default class PeoplePersonPersonalInformationRoute extends Route {
       include: [
         'mandatories.mandate.governing-body.is-time-specialization-of.administrative-unit.classification',
         'mandatories.contacts',
+        'agents-in-position',
       ].join(),
     });
+    const contacts = [];
```

Comment:
This sounds more like an array of positions?

```suggestion
    const positions = [];
```